src/traces/scattermapbox/attributes.js

```diff
@@ -26,6 +27,51 @@ module.exports = overrideAll({
     lon: scatterGeoAttrs.lon,
     lat: scatterGeoAttrs.lat,
 
+    cluster: {
+        enabled: {
+            valType: 'boolean',
+            role: 'info',
+            dflt: false,
+            description: 'Determines whether clustering is enabled or disabled.'
+        },
+        maxzoom: mapboxLayoutAtributes.maxzoom,
```

Comment:
`maxzoom` is inside `layers` and we also want to override the description.
So, this line should become:
```js
maxzoom: extendFlat({}, mapboxLayoutAtributes.layers.maxzoom, {
    description: [
        'Sets the maximum zoom level.',
        'At zoom levels equal to or greater than the maxzoom, the layer will be hidden.'
    ].join(' ')
}),
```